tfc-backup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+# tfc-backup.sh - Back up Terraform Cloud workspaces and variables to a Restic repository on Backblaze B2
+#
+# Dale Newby
+# SIL International
+# July 20, 2023
+
+# Required environment variables:
+# ATLAS_TOKEN        - Terraform Cloud access token
```

Comment:
Just opinion, but I don't like to see duplicated information since humans tend to not keep duplicated information in sync very well. But you're welcome to leave this documentation in two places if you want.